<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>309</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>First Source</string>
    </property>
    <widget class="QComboBox" name="comboBox_Source1_i">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Source1_o">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Second Source</string>
    </property>
    <widget class="QComboBox" name="comboBox_Source2_i">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Source2_o">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Important!!! Particular MIDI deivce can only be used once</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
